src/legacy/server/saved_objects/saved_objects_mixin.test.js

```diff
@@ -185,82 +185,6 @@ describe('Saved Objects Mixin', () => {
     });
   });
 
-  describe('Routes', () => {
-    it('should create 12 routes', () => {
-      savedObjectsMixin(mockKbnServer, mockServer);
-      expect(mockServer.route).toHaveBeenCalledTimes(12);
-    });
-    it('should add POST /api/saved_objects/_bulk_create', () => {
-      savedObjectsMixin(mockKbnServer, mockServer);
-      expect(mockServer.route).toHaveBeenCalledWith(
-        expect.objectContaining({ path: '/api/saved_objects/_bulk_create', method: 'POST' })
-      );
-    });
```

Comment:
This is already covered by the API integration tests, so I did not move these tests to core. Tell me if I should.